bitbots_vision/src/bitbots_vision/vision_modules/live_fcnn_03.py

```diff
@@ -5,7 +5,8 @@
 
 class FCNN03:
     """
-    Defines the FCNN neural network
+    Defines the FCNN neural network.
+    Tensorflow network definition of the FCNN03 model.
```

Comment:
Seems like the first line is uneccesary since the second line says the same and more?